app/assets/javascripts/components/SkipRuleSetFormField.jsx

```diff
@@ -33,7 +33,7 @@ class SkipRuleSetFormField extends React.Component {
           hide={this.props.hide}
           key={props.key || props.id}
           laterItems={this.state.laterItems}
-          namePrefix={`questioning[skip_rules_attributes][${index}]`}
+          namePrefix={`${this.state.type}skip_rules_attributes][${index}]`}
```

Comment:
Missing a `[` here after interpolation. I think this slides by CI b/c we don't have feature spec yet!

Reply:
oooh good catch! thank you!